src/components/TooltipMenuIcon/TooltipMenuIcon.tsx

```diff
@@ -0,0 +1,53 @@
+import { ComponentProps, ElementType, PropsWithChildren } from 'react';
+import clsx from 'clsx';
+import { extendTheme, ThemeProvider, Tooltip, TooltipProps } from '@chakra-ui/react';
+import './TooltipMenuIcon.scss';
+
+const theme = extendTheme({
+    zIndices: {
+        tooltip: 1800,
+    },
+});
+
+type AsElement = 'a' | 'button' | 'div';
+type TTooltipMenuIcon<T extends AsElement> = ComponentProps<T> & {
+    as: T;
+    disableHover?: boolean;
+    tooltipContent: string;
+    tooltipPosition?: TooltipProps['placement'];
+};
+
+const TooltipMenuIcon = <T extends AsElement>({
```

Comment:
nit: can you add a todo here to replace this with the component that will eventually come from deriv ui